.github/workflows/cypress-chrome.yml

```
@@ -0,0 +1,22 @@
+name: Cypress push and pull
+
+on: [push, pull_request]
+
+jobs:
+  cypress-run:
+    name: chrome
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: update test fixture
+        run: npm run pre:cy:genfix
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          build: npm run serve
```

I would hope the workflows timeout after 6hrs... they are not doing anything after serving

---

```suggestion
          start: npm run serve
```
I'm not 100% sure why you'd want to `build` and `start` separately, but AFAIK, you need `start` for Cypress to know "once this is done, I should start testing"

---

Thanks Oliver